### DISCUSSION TITLE
Detect keybinding for a specific command, and remap it to my extension's command

### DISCUSSION THREAD

**Discussion by mrigankpawagi** [microsoft/vscode-discussions#476] [Extension Development QnA] [answered]
I want to augment the behaviour of **editor.action.inlineSuggest.commit** through my extension. The default trigger for this command is the `tab` key, and so I mapped the `tab` key to my own command. This command then performs the extra functionality, and then calls `vscode.commands.executeCommand('editor.action.inlineSuggest.commit')`. The resulting behaviour is exactly what I've been looking for. 

However,  **editor.action.inlineSuggest.commit** need not have `tab` as its keybinding. To address this situation, I want to programmatically detect the keybinding for this command, and then map it to my extension's command. How can I achieve this? Any related references to documentation would be helpful (I wasn't able to find anything). Thanks a lot!

**Comment by isidorn** [comment, marked as answer]
Not currently possible programmatically detect a keybinding for a command.
